### PR TITLE
CCE fixes for better CLI handling

### LINKED ISSUE #146: Unify behavior of CCE CLI client
There some issues while using openstack client with the CCE extension while creating CCE nodes. This has to be reworked to achieve a unified user experience.

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: otcextensions/tests/unit/osclient/cce/v2/fakes.py [hunk 1]
```diff
@@ -85,13 +85,17 @@ def generate(cls):
                 'login': {
                     'sshKey': 'key-' + uuid.uuid4().hex,
                 },
-                'data_volumes': [
+                "rootVolume": {
+                    "type": "SATA",
+                    "size": random.randint(40, 100)
+                },
```
Comment (lego963) [inline otcextensions/tests/unit/osclient/cce/v2/fakes.py:91]: Change double quotes to single, pls

Comment (lego963) [inline otcextensions/tests/unit/osclient/cce/v2/fakes.py]: Change double quotes to single, pls

File: otcextensions/osclient/cce/v2/cluster_node.py [hunk 2]
```diff
@@ -143,143 +164,301 @@ def take_action(self, parsed_args):
...
+        )
+        parser.add_argument(
+            '--name',
+            metavar='<name>',
+            help=_('Name of the CCE node.\nThe clustername and a random '
+                   'number is used to create a node name by default.')
+        )
+        parser.add_argument(
+            '--annotation',
+            metavar='<key_name>=<value=name>',
```
Comment (lego963) [inline otcextensions/osclient/cce/v2/cluster_node.py]: `value_name`

File: examples/cce/create_cluster_node.py [hunk 3]
```diff
@@ -20,49 +20,67 @@
...
+            'eip': {
+                'iptype': '5_bgp',
+                'bandwidth': {
+                    'chargemode': 'traffic',
+                    'size': 10,
+                    'sharetype': 'PER'
+                }
+            }
+        },
+        'count': 2,
```
Comment (lego963) [inline examples/cce/create_cluster_node.py:75]: why did you change from `1` to `2`?

**Review by lego963 — CHANGES_REQUESTED** [review]

File: examples/cce/create_cluster_node.py (hunk 3 above)
Comment (tischrei) [reply examples/cce/create_cluster_node.py:75]: There are also 2 public IP's in this example. So I wanted to highlight that also 2 nodes can be requested.

File: otcextensions/tests/unit/osclient/cce/v2/fakes.py (hunk 1 above)
Comment (tischrei) [reply otcextensions/tests/unit/osclient/cce/v2/fakes.py]: done

File: otcextensions/osclient/cce/v2/cluster_node.py (hunk 2 above)
Comment (tischrei) [reply otcextensions/osclient/cce/v2/cluster_node.py]: done

File: otcextensions/osclient/cce/v2/cluster_node.py [hunk 4]
```diff
@@ -143,143 +164,301 @@ def take_action(self, parsed_args):
...
-                '* `SATA` = Common I/O \n'
-                '* `SAS` = High I/O \n'
-                '* `SSD` = Ultra-High I/O \n'
-                '`SIZE` is size in Gb.')
-        )
-        parser.add_argument(
-            '--root_volume',
-            metavar='<root_volume>',
+            '--floating-ip',
+            metavar='<floating-ip>',
```
Comment (lego963) [inline otcextensions/osclient/cce/v2/cluster_node.py]: `floating_ip`

Comment (tischrei) [reply otcextensions/osclient/cce/v2/cluster_node.py]: done

**Review by lego963 — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]
